examples/example_066.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
```

Comment:
@nicolaasuni if you find this block style more clean I can make a PR for it
Already converted all the blocks on the fork, so cherry-pick and PR is possible